static/css/stylesheet.css

```diff
@@ -58,30 +58,35 @@ a{
   padding: 10px;
 }
 
-.navbar .navbar-inner{
-  padding:0
-}
-
 .navbar {
   border: none;
   border-radius: 0;
   margin: 0;
-  display: table;
   width: 100%;
   background-color: #2c3e50;
+  display: flex;
 }
 
-.navbar-nav .nav ul{
-  width: 100%;
+.navbar button {
+  border: none;
+  background-color: Transparent;
 }
 
-.navbar .nav li {
-  width: calc(100vw/5);
-  display: table-cell;
-  float: none;
+.navbar button div {
+  width: 20px;
```

Comment:
yeahh... I just did some hacky stuff to try and make the navbar responsive 😛 changed the main navbar entries to flexboxes, but didn't mess with any of the dropdown styling. I can mess around w/ dropdown styling to fix this